8Knot/pages/contributions/visualizations/cntrb_pr_assignment.py

```diff
@@ -372,5 +372,10 @@ def pr_assignment(df, start_date, end_date, contrib):
         (df_in_range["assignment_action"] == "assigned") & (df_in_range["assign_date"] <= end_date)
     ]
 
-    # return the different of assignments and unassignments
-    return df_assigned.shape[0] - df_unassign.shape[0]
+    # the different of assignments and unassignments
+    assign_value = df_assigned.shape[0] - df_unassign.shape[0]
+
+    # prevent negative assignments
+    assign_value = 0 if assign_value < 0 else assign_value
```

Comment:
why can this be negative in the given range?

Reply:
Ill be honest its been a while since I implemented this/ I dont think I fully knew at the time either. I think it is if there are assignments before the github apis start point for really large repos (like rust in the example) and then they are unassignments of those it ends up negative